packages/components/src/BaseStyleSheet.scss

```diff
@@ -25,7 +25,7 @@ body {
   padding: 0;
   overscroll-behavior: none;
   -ms-scroll-chaining: none;
-  scrollbar-color: rgba($white, 0.1) rgba($black, 0.1); //applies to firefox only
+  scrollbar-color: auto; //applies to firefox only
```

Comment:
I'm a little confused why you made this auto? When dragging in firefox now the scrollbar is red.

https://user-images.githubusercontent.com/4505624/131732375-4b4e5a48-bfc5-481c-9969-1e4da839dd4a.mp4

Reply:
Oh firefox, my special little child. I guess it goes wild on ubunutu, didn't realize that.

I'll revert that I guess.